scipy/interpolate/ndgriddata.py

```diff
@@ -92,12 +92,11 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
 
     Parameters
     ----------
-    points : ndarray of floats, shape (n, D)
-        Data point coordinates. Can either be an array of
-        shape (n, D), or a tuple of `ndim` arrays.
-    values : ndarray of float or complex, shape (n,)
+    points : 2-D ndarray of floats with shape (n, D), or length D tuple of 1-D ndarrays with shape (n,).
+        Data point coordinates. 
+    values : 1-D ndarray of float or complex with shape (n,)
```

Comment:
It doesn't have to be 1D, for `ndim` input each data point is a `ndim-1` array.

Reply:
I can bring this PR up-to-date, but I'm a little confused about this comment. The set of points is a 2D array of all points, which can each be D dimensional. So the values should be a 1 dimensional array of length n, correct? Why is this wrong?